### DISCUSSION TITLE
ADMINISTRATOR NOTICE: The query to URIBL was blocked

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2354] [Help with using Postal]
Hello,

I have a problem with the OUTBOUND SPAM THRESHOLD function.

Everything is installed correctly but I have the following message in the analysis of the message:

**URIBL_BLOCKED

ADMINISTRATOR NOTICE: The query to URIBL was blocked. See http://wiki.apache.org/spamassassin/DnsBlocklists#dnsbl-block for more information. [URIs: xxxx-xxx.com]**

I edit /etc/resolv.conf to change DNS for Google : 
`nameserver 8.8.8.8
nameserver 8.8.4.4`

I still get the error, has anyone had this problem before and found a solution?

Regards

**Comment by willpower232** [comment]
where are you seeing this error message? can you share a screenshot?

**Reply by SaasX** [reply]
![screen](https://user-images.githubusercontent.com/79661655/217578550-610372f4-9c1a-4a3a-9ab7-7270a4d53831.png)

**Reply by willpower232** [reply]
I guess they don't like you or your server provider at the minute judging by their answer

![image](https://user-images.githubusercontent.com/1619102/217599337-7cda05b6-f23d-4bdb-ab29-18f4db0e4907.png)

**Comment by SaasX** [comment]
https://support.cpanel.net/hc/en-us/articles/360053079473-SpamAssassin-headers-show-URIBL-Blocked-error-when-Exim-uses-RBLs
